saleor/order/actions.py

```diff
@@ -236,8 +243,115 @@ def automatically_fulfill_digital_lines(order: "Order"):
         FulfillmentLine.objects.create(
             fulfillment=fulfillment, order_line=line, quantity=quantity
         )
-        fulfill_order_line(order_line=line, quantity=quantity)
+        warehouse_pk = line.allocations.first().stock.warehouse.pk  # type: ignore
+        fulfill_order_line(
+            order_line=line, quantity=quantity, warehouse_pk=warehouse_pk
+        )
     emails.send_fulfillment_confirmation_to_customer(
         order, fulfillment, user=order.user
     )
     update_order_status(order)
+
+
+def _create_fulfillment(
```

Comment:
Does it create fulfillment? I think a better name would be `create_fulfillment_lines` as it returns a list of lines.